src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CourseActivity.kt

```diff
@@ -90,7 +90,7 @@ data class CourseActivity(
   val scheduleEndDate: LocalDate? = null,
 
   @OneToMany(mappedBy = "courseActivity", cascade = [CascadeType.ALL], fetch = FetchType.LAZY)
-  var payRates: List<CourseActivityPayRate>? = null,
+  var payRates: MutableList<CourseActivityPayRate> = mutableListOf(),
```

Comment:
Copied the same pattern as elsewhere in this app - relationships between entities need to be mutable.